sky/engine/BUILD.gn

```diff
@@ -35,6 +35,10 @@ config("non_test_config") {
 }
 
 group("engine") {
+  public_configs = [
```

Comment:
The problem is that without Wno-inconsistent-missing-override, clients of the engine can't include any of the engine's headers because they are inconsistent about missing overrides.  (This is actually the problem that caused me to make it a public config.)